src/core/utils/format-amount.ts

```diff
@@ -0,0 +1,24 @@
+import { BigNumber } from 'bignumber.js';
```

Comment:
These are useful functions, but I would simplify things.

Let's create a `Amount` component, that receives these props:
```ts
interface IExternalProps {
    amount: BigNumber | number | string
    currency: string,
    convertTo?: string // if this is present will convert amount to this currency
}
```
So instead of all these functions we can use this component. (Everywhere we need to display an amount, we have the values so there's no need to have specialised function for different kind of objects.